docs/modules/layout.md

```diff
@@ -0,0 +1,20 @@
+## Project layout
+### Module layout
```

Comment:
Since 8.x-2.x branch is the default and the one which the users will download I feel mentioning branch here would not be needed from users point of view. Also since we will be merging mkdocs branch to 8.x-2.x branch, is it ok if we add the branch details as a footnote here?

Reply:
OK, makes sense since we'll be merging the doc on the main 8.2 branch.